interface/themes/style_light.css

```diff
@@ -697,7 +697,7 @@ tr.odd, td.even {
 .css_button span, input[type="button"] span, button span {
   display: block;
 }
-.css_button[href*="deleter"], input[type="button"][href*="deleter"], button[href*="deleter"] {
+.css_button[href*="deleter"], input[type="button"][href*="deleter"], input[type="button"][class="deleter"], button[href*="deleter"] {
```

Comment:
Allows for the red styling of 'Cancel' buttons. There are other areas in the codebase where this can be used.

Reply:
Will file an issue for this and fix it.